components.yaml

```diff
@@ -39,7 +39,7 @@ components:
     url: https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller
     commit: f40f85bc20f361aeeac7b4b04aa6d289c8e130f5
     branch: main
-    update-policy: static
+    update-policy: tagged
```

Comment:
multus is static here, and the values there are different than the ones on 0.89
is it safe to change multus-dynamic-networks to tagged ?
or we need at least to change manually also multus ?

we can check if lanes are passing with just changing multus-dynamic-networks - will create PR for that